src/Common/src/System/Net/NTAuthentication.Common.cs

```diff
@@ -292,7 +292,8 @@ internal byte[] GetOutgoingBlob(byte[] incomingBlob, bool throwOnError, out Secu
             }
 
             // The return value will tell us correctly if the handshake is over or not
-            if (statusCode.ErrorCode == SecurityStatusPalErrorCode.OK)
+            if (statusCode.ErrorCode == SecurityStatusPalErrorCode.OK
+                || statusCode.ErrorCode == SecurityStatusPalErrorCode.CompleteNeeded) // Server only
```

Comment:
Is there any danger in treating this error code as success when this is actually the client?  I'm just wondering if this should be:
```C#
if (statusCode.ErrorCode == SecurityStatusPalErrorCode.OK ||
    (_isServer && statusCode.ErrorCode == SecurityStatusPalErrorCode.CompleteNeeded))
```

Reply:
I can change it, but this isn't a state you should ever see on the client.